### ISSUE TITLE
test failure on 0.6-pre.beta.105

### ISSUE THREAD

**Issue by PaulSoderlind** [JuliaIO/Formatting.jl#36]
Hi,
on Windows 7 (64), I see the following error from Pkg.test("Formatting")

ERROR: LoadError: LoadError: MethodError: no method matching (::##1#2)(::Float64)
The applicable method may be too new: running in world age 21681, while current world is 21682.
Closest candidates are:
  #1(::Any) at D:\Julia\Packages\v0.6\Formatting\test\cformat.jl:13 (method too new to be called from this world context.)
Stacktrace:
 [1] test_equality() at D:\Julia\Packages\v0.6\Formatting\test\cformat.jl:18
 [2] include_from_node1(::String) at .\loading.jl:539
 [3] include(::String) at .\sysimg.jl:14
 [4] include_from_node1(::String) at .\loading.jl:539
 [5] include(::String) at .\sysimg.jl:14
 [6] process_options(::Base.JLOptions) at .\client.jl:305
 [7] _start() at .\client.jl:371

My own code suggests that fmt() works, but sprintf1() does not.

**Comment by JackDevine** [comment]
Hi,

I can duplicate this on OSX and on Ubuntu (with Julia0.6). Something that might also be related:
```julia
format(1.23; precision=2)
```

```
MethodError: no method matching sprintf_JS4yZg!!(::Float64)
The applicable method may be too new: running in world age 24206, while current world is 24207.
Closest candidates are:
  sprintf_JS4yZg!!(::Any) at /home/devja964/.julia/v0.6/Formatting/src/cformat.jl:24 (method too new to be called from this world context.)

Stacktrace:
 [1] #format#2(::Int64, ::Int64, ::Bool, ::Bool, ::Bool, ::Bool, ::Bool, ::Bool, ::Bool, ::Bool, ::Bool, ::String, ::String, ::Int64, ::Int64, ::String, ::Symbol, ::String, ::Formatting.#format, ::Float64) at /home/devja964/.julia/v0.6/Formatting/src/cformat.jl:257
 [2] (::Formatting.#kw##format)(::Array{Any,1}, ::Formatting.#format, ::Float64) at ./<missing>:0


```
 If I run the function again, then I get `"1.23"` as expected.
```julia
julia> Pkg.status("Formatting")
 - Formatting                    0.2.1

```

**Comment by swadey** [comment]
same issue with sprintf1

**Comment by ScottPJones** [comment]
I believe this can be closed now - it was fixed by #30.
